src/battle_anim_effects_2.c

```diff
@@ -78,7 +78,7 @@ static void AnimTask_SplashStep(u8);
 static void AnimTask_GrowAndShrinkStep(u8);
 static void ThrashMoveMonStep(u8);
 static void ThrashMoveMon(u8);
-static void AnimTask_SketchDrawMon(u8);
+static void AnimTask_SketchDrawMonStep(u8);
```

Comment:
There is no reason to add the `Step` prefix if it's a single task without any additional steps.

Reply:
Sure. I just need a way to distinguish it from `AnimTask_SketchDrawMon`. I've been avoiding giving everything in the battle scripts file an `Init` prefix